benchmark/qwen3-asr-webgpu/results/1.7b-memory/memwatch.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Sample every 0.5 s while a harness run is in flight: the RSS of every headless_shell / chrome
+# process by type, plus the system-wide MemAvailable (kB). On the GB10 (unified memory,
+# nvidia-smi reports N/A, Vulkan device-local buffers are not mapped into the process) the
+# drop in MemAvailable is the only number that includes the GPU allocations.
+# usage: memwatch.sh <outfile> &  ... run the page ...  kill %1 ; then: memwatch_report.py <outfile>
+OUT="$1"
+: > "$OUT"
+while true; do
+  t=$(date +%s.%N)
+  ps -eo rss,args 2>/dev/null | grep -E 'headless_shell|chrome' | grep -v grep | awk -v t="$t" '{
```

Shellcheck (0.11.0)

[info] Consider using pgrep instead of grepping ps output.

(SC2009)